src/sdl2/hint.rs

```diff
@@ -3,6 +3,27 @@ use sys::hint as ll;
 use std::ptr;
 use libc::c_char;
 
+const VIDEO_MINIMIZE_ON_FOCUS_LOST: &'static str = "SDL_VIDEO_MINIMIZE_ON_FOCUS_LOST";
+
+/// Sets the SDL_HINT_VIDEO_MINIMIZE_ON_FOCUS_LOSS hint.
+///
+/// # Default
+/// By default a SDL_Window is minimized if it loses key focus when in fullscreen mode.
```

Comment:
You don't have to. One link to the official document is enough, users are expected to go look at the official SDL2 documentation if they need additional information.